scripts/start_super_protocol.sh

```
@@ -341,58 +353,58 @@
 
 prepare_gpus_for_vfio() {
     local gpu_ids=("$@")
-    
+
     # Detect NVSwitch devices
     local nvswitch_ids=($(lspci -mm -n -d 10de:22a3 | cut -d' ' -f1))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)